pkg/provider/util.go

```diff
@@ -31,6 +31,19 @@ func hasComputedValue(obj *unstructured.Unstructured) bool {
 			if field, isMap := v.(map[string]interface{}); isMap {
 				objects = append(objects, field)
 			}
+			if field, isSlice := v.([]interface{}); isSlice {
```

Comment:
I'm confused about this code generally. This doesn't handle arbitrarily-nested slices, right?

Reply:
No, probably not. I'll take another look